Jenkinsfile

```diff
@@ -8,6 +8,19 @@ pipeline {
     }
 
     stages {
+        stage('Common Module Unit Tests') {
+            steps {
+                dir('common') {
+                    sh label: 'Installing dependencies', script: 'pipenv install --dev --deploy --ignore-pipfile'
+                    sh label: 'Running unit tests', script: 'pipenv run unittests-cov'
+                    sh label: 'Displaying code coverage report', script: 'pipenv run coverage-report'
+                    sh label: 'Exporting code coverage report', script: 'pipenv run coverage-report-xml'
+                    cobertura coberturaReportFile: '**/coverage.xml'
```

Comment:
Looking at the Jenkins build for this branch/PR, it appears that it's picking up the same coverage report (for the MHS) twice, rather than reporting coverage for the common module.

Reply:
Updated the Jenkins file so that it now has a single coverage stage which combines both reports